support-frontend/app/views/contributions.scala.html

```diff
@@ -39,17 +39,15 @@
       @user.publicFields.displayName.map { displayName =>
         displayName: "@displayName",
       }
-      @for(fields <- user.privateFields) {
-        @for(firstName <- fields.firstName; lastName <- fields.secondName) {
-          firstName: "@firstName",
-          lastName: "@lastName",
-        }
-        @for(address4 <- fields.billingAddress4 orElse fields.address4) {
-          address4: "@address4",
-        }
-        @for(country <- fields.billingCountry orElse fields.country) {
-          country: "@country"
-        }
+      @for(firstName <- user.privateFields.firstName; lastName <- user.privateFields.secondName) {
```

Comment:
`User` from identity-model is now being used (instead of `User` from identity-play-auth); in particular, the `privateFields` field is no longer modelled as an option (which is why a nested for comprehension is no longer necessary).